src/esp/core/test/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+find_package(Corrade REQUIRED Utility)
+
+corrade_add_test(coreLoggingTest LoggingTest.cpp LIBRARIES core Corrade::Utility)
```

Comment:
Linking `Corrade::Utility` explicitly won't hurt, it's just done implicitly (it's a dependency of `TestSuite`).